src/components/ModalPayment/ModalAmount/ModalAmount.tsx

```diff
@@ -44,28 +46,64 @@ export const Modal = (props: Props) => {
     return r.test(value);
   };
 
-  const buttonAmounts = [
-    { value: '10', text: '$10' },
-    { value: '25', text: '$25' },
-    { value: '50', text: '$50' },
-    { value: '100', text: '$100' },
-  ];
+  const buttonAmounts =
+    modalView === ModalPaymentTypes.modalPages.light_up_chinatown
+      ? [
+          { value: '25', text: '$25' },
+          { value: '45', text: '$45' },
+          { value: '150', text: '$150' },
+          { value: '300', text: '$300' },
+        ]
+      : [
+          { value: '10', text: '$10' },
+          { value: '25', text: '$25' },
+          { value: '50', text: '$50' },
+          { value: '100', text: '$100' },
+        ];
```

Comment:
Uh don't rely on my typescript syntax since it's not the main point but the following idea seems less error prone
```

const buttonAmountsArray = modalView === ModalPaymentTypes.modalPages.light_up_chinatown
	? [25, 45, 150, 300]
	: [10, 25, 50, 100]
const buttonAmounts = Array.map(buttonAmountsArray, x => { value: x.toString(), text: '$' + x.toString() });
```